README.md

```diff
@@ -22,6 +23,24 @@ lm.emit_metric(name='logmet.test.2', value=2)
 lm.emit_metric(name='logmet.test.3', value=3)
 ```
 
+## Sending logs
+```python
+import logmet
+
+lm = logmet.Logmet(
+    logmet_host='logs.opvis.bluemix.net',
+    logmet_port=9091,
```

Comment:
I think I will deprecate these keywords in favor of logs_host, logs_port, and metrics_host, metrics_port. That way a user can use the same Logmet object to send either or both.